app/assets/javascripts/uploadcare/widget/tabs/camera-tab.coffee

```diff
@@ -166,17 +166,18 @@ uploadcare.namespace 'widget.tabs', (ns) ->
       @__setState('recording')
 
       @__chunks = []
-      __recorderOptions = {
-        mimeType: 'video/webm'
-      }
+      __recorderOptions = {}
 
       if @settings.audioBitsPerSecond != null
         __recorderOptions.audioBitsPerSecond = @settings.audioBitsPerSecond
 
       if @settings.videoBitsPerSecond != null
         __recorderOptions.videoBitsPerSecond = @settings.videoBitsPerSecond
 
-      @__recorder = new @MediaRecorder(@__stream, __recorderOptions)
+      if Object.keys(__recorderOptions).length != 0
+        @__recorder = new @MediaRecorder(@__stream, __recorderOptions)
+      else
+        @__recorder = new @MediaRecorder(@__stream)
```

Comment:
Restore the original safe code since [MDN says there are browsers](https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder/MediaRecorder#Browser_compatibility) who don't support the second argument.